build_scripts/post_audit_issues.py

```
@@ -0,0 +1,150 @@
+        capture_output=True, text=True,
+    )
+    if result.returncode != 0:
+        print(f'WARNING: could not fetch existing issues: {result.stderr}', file=sys.stderr)
+        _existing_titles = set()
+    else:
+        _existing_titles = {i['title'] for i in json.loads(result.stdout or '[]')}
+    return _existing_titles
+
+
+def _ensure_label() -> None:
+    subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)